### DISCUSSION TITLE
[Lib] beParticles

### DISCUSSION THREAD

**Discussion by paladin-t** [paladin-t/bitty#14] [Libs And Tips]
Particle system implemented in pure Lua for [Bitty Engine](https://github.com/paladin-t/bitty/). It is ported from [MaxwellDexter/pico-ps](https://github.com/MaxwellDexter/pico-ps).

![](https://raw.githubusercontent.com/paladin-t/beparticles/main/imgs/beParticles1.gif)
![](https://raw.githubusercontent.com/paladin-t/beparticles/main/imgs/beParticles2.gif)
![](https://raw.githubusercontent.com/paladin-t/beparticles/main/imgs/beParticles3.gif)

Latest version:
* Repository: [github](https://github.com/paladin-t/beparticles)

**Comment by xjxxjx1017** [comment]
This modified version below added width / height parameters, bitty engine's sprite and texture rendering support:
attachment here: [pico-ps.zip](https://github.com/paladin-t/bitty/files/7616176/pico-ps.zip)
